__tests__/test-utils/command-build.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const spawn = require('cross-spawn');
+
+module.exports = function commandBuild(argv, { env, stdio = 'inherit' } = {}) {
+  env = Object.assign({}, process.env, { NODE_ENV: '', BABEL_ENV: '' }, env);
```

Comment:
Why do we need to add the empty string defaults?

Reply:
This is a way to unset the parent processes env variable, as the `env` object being passed to child process would get stringified vars (as a side effect `undefined` would not work). 

If this looks confusing I can manually add `delete env[NODE_ENV];`.